.github/workflows/close-stale.yml

```diff
@@ -20,8 +20,8 @@ jobs:
     - uses: actions/stale@v3
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
-        stale-issue-message: 'This issue is stale because it has been open 30 days with no activity. Remove stale label / comment or this will be closed in 5 days.'
+        stale-issue-message: 'This issue has had no activity in the last 30 days. Please add a comment if you are still impacted by this issue or are working on a patch. This issue will be automatically closed if no comments are added within 7 days.'
         days-before-stale: 30
-        days-before-close: 5
+        days-before-close: 7
         stale-issue-label: 'stale-closing-soon'
-        exempt-issue-labels: 'T: Feature Request'
+        exempt-issue-labels: 'T: Feature Request,Bug: Confirmed !,no-locking'
```

Comment:
I do not know how I could test this. It wasn't obvious to me what syntax would be used with spaces and punctuation in the label names. I followed the existing syntax from their action's repo, but it might be necessary to escape certain characters.

`exempt-pr-labels: 'awaiting-approval,work-in-progress'`

Reply:
Also, I know there is some disagreement on whether "Bug: Confirmed !" should be reason enough to keep something open, although I think we do need some way to keep particular important issues open without a nag routine.

Reply:
Relying on labels (other than feature request) is sketchy, because labels like "bug confirmed" are applied in a very spotty fashion. Also, such labels are almost never removed, so they just become a random reason that some old issues will never close.

If some issue is supposed to be kept open after a long period of quiet, the author of the issue should take responsibility to keep it alive. Issues in themselves have little to no value if they aren't being actively worked on. So, I think it's good that open issues of all kinds are nagged, closed, and locked so that what's left over is mostly current and relevant.

It would be nice if there were only a few new issues a day. The current volume of new issues makes it quite impossible to attend to them all. At the current rate I can spend every waking hour working on Marlin issues and the queue will never get any smaller.